x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ssl/SSLService.java

```diff
@@ -589,36 +590,18 @@ synchronized void reload() {
 
         private void reloadSslContext() {
             try {
-                X509ExtendedKeyManager loadedKeyManager = Optional.ofNullable(keyConfig.createKeyManager(env)).
-                    orElse(getEmptyKeyManager());
-                X509ExtendedTrustManager loadedTrustManager = Optional.ofNullable(trustConfig.createTrustManager(env)).
-                    orElse(getEmptyTrustManager());
+                X509ExtendedKeyManager loadedKeyManager = keyConfig.createKeyManager(env);
+                X509ExtendedTrustManager loadedTrustManager = trustConfig.createTrustManager(env);
```

Comment:
We'd never call `getEmptyTrustManager` as none of the implementations of `createTrustManager` returns null. As for an emtpyKeyManager, this could only happen for KeyConfig NONE - so I moved this there